connector_test.go

```diff
@@ -403,29 +370,28 @@ func TestCgroupNsByNamespacePath(t *testing.T) {
 	configtemplate2 := fmt.Sprintf(cgroupTemplate, containerNamePrefix2, namespacePath)
 	connector2, _ := getConnector(t, configtemplate2)
 
-	container2, err := connector2.Deploy(context.Background(), "quay.io/tsebastiani/arcaflow-engine-deployer-podman-test:latest")
-	if err != nil {
-		t.Fatalf(err.Error())
-	}
+	container2, err := connector2.Deploy(
+		context.Background(),
+		"quay.io/tsebastiani/arcaflow-engine-deployer-podman-test:latest")
+	assert.NoError(t, err)
 
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
 		var containerInput = []byte("sleep 5\n")
-		if _, err := container2.Write(containerInput); err != nil {
-			t.Errorf(err.Error())
-			return
-		}
+		_, err := container2.Write(containerInput)
+		assert.NoError(t, err)
 	}()
 
 	ns1 := tests.GetPodmanPsNsWithFormat(logger, config.Podman.Path, container1.ID(), "{{.CGROUPNS}}")
 	ns2 := tests.GetPodmanPsNsWithFormat(logger, config.Podman.Path, container1.ID(), "{{.CGROUPNS}}")
-	if ns1 != ns2 {
-		t.Errorf("namespace1: %s and namespace2: %s do not match, failing", ns1, ns2)
-	} else {
-		logger.Debugf("container 1 namespace: %s, container 2 namespace: %s, they're the same!", ns1, ns2)
-		logger.Debugf("Container 2 joined the namespace via namespace path: %s", namespacePath)
-	}
+	//if ns1 != ns2 {
+	//	t.Errorf("namespace1: %s and namespace2: %s do not match, failing", ns1, ns2)
+	//} else {
+	//	logger.Debugf("container 1 namespace: %s, container 2 namespace: %s, they're the same!", ns1, ns2)
+	//	logger.Debugf("Container 2 joined the namespace via namespace path: %s", namespacePath)
+	//}
```

Comment:
Stale comments.
I will say that the old comments are more explicit than the concise assertion.